contents/docs/data/identify.mdx

```diff
@@ -105,7 +105,7 @@ curl -v -L --header "Content-Type: application/json" -d '{
 There are two requirements when assigning an `alias_id`:
 
 1. It cannot be associated with more than one `distinct_id`.
-2. The `alias_id` **must not** be the `distinct_id` of any other user. For example, assume we previously identified a user with `distinct_id_one` in a different session and device e.g., in [posthog-js](https://posthog.com/docs/libraries/js) using `posthog.identify('distinct_id_one')`. It is not possible to use `distinct_id_one` as an alias for any another distinct ID:
+2. The `alias_id` **must not** be the `distinct_id` of any other identified user. For example, assume we previously identified a user with `distinct_id_one` in a different session and device e.g., in [posthog-js](https://posthog.com/docs/libraries/js) using `posthog.identify('distinct_id_one')`. It is not possible to use `distinct_id_one` as an alias for any another distinct ID:
```

Comment:
this is not helping me understand it better 😅 

2. must not have been used as `distinct_id` for these events `$identify`, `$create_alias`, `$merge_dangerously` previously.